GA-worksheet.md

```diff
@@ -25,45 +25,45 @@
 
 ##Project #2 Part #2: Style the Header with CSS
 
-1. What advantages does the `<link>` tag have over the `<style>` tag?
+1. What advantages does the `<link>` tag have over the `<style>` tag? Instead of putting your CSS directly in the html file and taking up space-like the <style> tag, the <link> tag, is a connection from your html to a css, that has all he properties and attributes you want. It's advantageous because it makes your html shorter and easier to read-something especially convenient when you have dozens of style attributes.
 
-2. What is normalize.css?
+2. What is normalize.css? it's a "thing" that basically renders the elements in your page consistent across all browsers and wherever your page is being seen, so it doesn't look messy.
 
-3. Why do we need to add a padding property to the `<ul>` tag?
+3. Why do we need to add a padding property to the `<ul>` tag? to clear an area aroud nd the/a element and center the elements better.
```

Comment:
In the dash tutorial, the `text-align: center' in the header is actually what makes it centered.
There is default padding on the left for unordered lists to accommodate the bullet points, so adding padding to all sides makes this particular ul centered.